index.html

```diff
@@ -0,0 +1,204 @@
+              <p class="card-text">
+                We will teach you how to design and become a professional designer.
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/html-css-js.png" class="card-img-top" alt="html-css-js">
+            <div class="card-body">
+              <h5 class="card-title">Web Development Basics</h5>
+              <p class="card-text">
+                you will learn web development basics which are HTML, CSS, and JS
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/react-redux.jpg" class="card-img-top" alt="react-redux">
+            <div class="card-body">
+              <h5 class="card-title">ReactJS & Redux</h5>
+              <p class="card-text">
+                We will teach you one of popular frameworks ReactJS and Redux to handle states
+                to make your work easer and professional
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/node-mongo.png" class="card-img-top" alt="Ruby & Ruby on rails">
+            <div class="card-body pt-0">
+              <h5 class="card-title">Ruby On Rails</h5>
+              <p class="card-text">
+                Ruby on Rails makes it much easier and more fun. 
+                It includes everything you need to build fantastic applications,
+              </p>
+            </div>
+          </div>
+        </div>
+        <a href="#" class="h4 mt-5">
+          SEE WHOLE PROGRAM
+        </a>
+      </div>
+    </section>
+    <!-- Start Featured Teachers -->
+    <section class="title featured-teachers container text-center">
+      <h2 class="mb-5 mt-5">
+        Featured Teachers
+      </h2>
+      <div id="teachers" class="row">
+
+      </div>
+    </section>
+    <!-- Start Partner -->
+    <section class="partner container-fluid py-4 mb-5">
+      <h2 class="title text-center pt-3 mb-5">
+        Partner
+      </h2>
+      <div class="d-flex flex-wrap justify-content-around align-items-center">
+        <div class="img-container mt-2">
+          <img src="./imgs/mozilla.png" alt="mozilla" class="w-100"/>
+        </div>
+        <div class="img-container mt-2">
+          <img src="./imgs/line-friends.png" alt="line-friends" class="w-100"/>
+        </div>
+        <div class="img-container mt-2">
+          <img src="./imgs/googlekorea.png" alt="googlekorea" class="w-100"/>
+        </div>
+        <div class="img-container mt-2">
+          <img src="./imgs/airbnb.png" alt="airbnb" class="w-100"/>
+        </div>
+      </div>
```

Comment:
- [ ] In the mobile version, the layout of the speakers is wrong: you have one column and it must be two, compare it to the original design.

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/132547399-7e8a0b87-9b2d-42a1-b915-cbb1e04de7c1.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/132547455-c08bb20d-2638-4721-9463-282fefb14366.png)

Reply:
Nice! Good job